Formula/g/gnmic.rb

```diff
@@ -0,0 +1,28 @@
+class Gnmic < Formula
+  desc "GNMI CLI client and collector"
+  homepage "https://gnmic.openconfig.net"
+  url "https://github.com/openconfig/gnmic.git",
+      tag:      "v0.36.2",
+      revision: "a7844a6d7d47c302df1fabb845dd22e4e4b70e22"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/openconfig/gnmic/pkg/app.version=#{version}
+      -X github.com/openconfig/gnmic/pkg/app.commit=#{Utils.git_head}
```

Comment:
there is no need to do git checkout, either omit it or use `#{tap.user}`

Reply:
I use git so we can use the `Utils.git_head` function for specifying the commit information. it is displayed in the app version information.

Reply:
Where would I use tap.user, wouldn't the tap be homebrew-core? The `-X` variable configuration is for the package which needs to be specified as the full import path.